app/models/favorite.rb

```diff
@@ -0,0 +1,5 @@
+class Favorite < ActiveRecord::Base
+  validates_uniqueness_of :user_id, scope: :skatepark_id
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.